### ISSUE TITLE
As a user, I need to be able to open the platform in browsers other than chrome

### ISSUE THREAD

**Issue by LisaLNelson** [openopps/openopps-platform#1392]
Several users have expressed problems with opening the platform in various browsers.  Here is the message from Terry Lynn Sales CBP/DHS:

Darn it!  Tried Firefox, Edge, IE and Chrome.  Finally got it to come up in Chrome.  This might limit your respondents if Chrome is the only browser they open in.  I used to be able to open in IE.  Recent change?  

Another message from Johnothan Rears Data Manager/USDA (long time user) cannot open the platform in Firefox or IE and he used to be able to. He also said he has issues with Chrome as well. He can read the posts but you cannot  respond.

Safari does not seem to be working quite right either.  The site will open and tasks appear but not all of them open. For example: [Seeking Content Writer's for DigitalGov blog](url) opens to a blank page.

![screen shot 2016-06-17 at 1 16 26 pm](https://cloud.githubusercontent.com/assets/9039852/16158910/bb68a1fe-348d-11e6-80a4-5d1fedab1e09.png)

**Comment by jrears** [comment]
Not sure what happened to this platform since most of these things were fully functional not too long ago in Internet Explorer 11, Firefox, and Chrome. These browsers may have changed some since then, but really doubt they would break this platform... :-(

My current browser test set:

Chrome Version 51.0.2704.84 m
Firefox Version 40.0.3
Internet Explorer Version 11.0.9600.18314 Update 11.0.31

P.S. Good page coding to me supports standards and will thus be at least usable on older platforms and browsers. And since in an Enterprise Environment we cannot typically control or change what browsers or other applications we must use, flexibility should be focused on more. I would also suggest less scripting and coding and more static html and css forms OR use a good standard CMS platform like Drupal, Joomla, or other easily obtainable one.

**Comment by matthewsklar** [comment]
I replicated this error in Microsoft Edge when opening tasks. I run Windows 10 and this error did not occur in Chrome or Firefox. Whenever I would attempt to view a task in Edge a syntax error would be thrown. I traced the error to line 1454 of node_modules/underscore/underscore.js 
`var render = new Function(settings.variable || 'obj', '_', source);`
I will continue investigating.

**Comment by ultrasaurus** [comment]
@jrears we upgraded many of the dependent libraries recently, and should have done a lot more testing. Our release process was accelerated dues to resolving an unrelated security concern.  Apologies for the disruption.  

@matthewsklar thanks for looking into this.  I've set up Sauce Labs so that I can reproduce the issue, and will investigate with @jeremiak tomorrow am.

**Comment by ultrasaurus** [comment]
It looks like the issue might be with the jsdom package.  I can repro the issue on Windows 10, IE 11, using Sauce Labs and a local tunnel.  Here's the line where it fails:

![image](https://cloud.githubusercontent.com/assets/41906/16528105/91fccb28-3f6f-11e6-87f4-ce891a65f102.png)

**Comment by rogeruiz** [comment]
Looks like this is tied to `jsdom` not being compiled automatically for `es5`. Maybe we will need to use an older version of `jsdom`? https://github.com/tmpvar/jsdom#install

**Comment by rogeruiz** [comment]
@ultrasaurus It seems like having `babelify` / `browserify` transform files in a targeted way within the `node_modules` directory can prevent this from happening in the future. Here's the documentation for doing that using `babelify` and `browserify` respectively.

https://github.com/babel/babelify#why-arent-files-in-node_modules-being-transformed
https://github.com/substack/node-browserify#btransformtr-opts

**Comment by ultrasaurus** [comment]
task details page still fails:

![image](https://cloud.githubusercontent.com/assets/41906/16690301/0ecefa60-44dc-11e6-95a7-b9384cf20c74.png)

Here's the syntax error:
![image](https://cloud.githubusercontent.com/assets/41906/16690687/e8efa8a6-44dd-11e6-90cb-c12c2f432464.png)

**Comment by rogeruiz** [comment]
On further inspection of the code in that snippet: It's coming from `underscore` [here](https://github.com/jashkenas/underscore/blob/master/underscore.js#L1522-L1536). There are a number of issues that this could be related to in the `underscore` repository.
- jashkenas/underscore#818
- jashkenas/underscore#666
- jashkenas/underscore#1646

The second issue above ☝️ shows the most promise. It looks like we have a syntax issue in one of our client-side templates that uses `_.template(/*breaking code here... */)`. I'll be able to better track this down later this evening.

**Comment by LisaLNelson** [comment]
More issues reported today:

1. Bryan Jackson - I registered on Open Opportunities site and undated my profile. I could not upload a photo. Also the sponsor’s photos have the same black circle with a line through it as my photo has. I am assuming the site should be displaying the sponsor’s photo.
 
Additionally, when I click on one of the opportunities listed, a Openopps page opens but the page has no information. It is like opening a blank template.

2. Chenda Conklin -  In attempting to view the below Open Opportunity for the White House (pictured), when I click on it all I receive is a blank page (pictured). I am on a gov’t computer running Windows, Office 16, IE 11.
 
Also, the email address openopps-platform@googlegroups.com is bouncing (see attached). I tried 3 times.
